doc/README.md

```diff
@@ -0,0 +1,182 @@
+# Lightning Talks App
+
+## Introduction
+
+This is fullstack web project includs both [fontend](../client) & [backend](../server/) applications written in Typescript.
+Frontend based on React. Backend based on Express.
+
+## Quick Start
+
+After you pulled latest code from `master` branch.
+
+### 1. Install dependencies
+
+```bash
+# install dependencies of base, like eslint plugins for all projects
+yarn
+
+# install frontend related dependencies
+cd client
+yarn
+
+# install backend related dependencies
+cd ../server
+yarn
+```
+
+### 2. Setup database
+```bash
+# pull the database image, assume you have installed docker
+docker pull postgres:15
+
+# create environment variable file
+cp ./server/.env.sample ./server/.env
+# change the db config as you prefer
+vi ./server/.env
+
+# start db by docker compose
+yarn start:db
+
+cd ./server
+# deploy db tables and initl data with seed file.
+yarn migrate:deploy
+# deploy db tables and initl data with seed file and generate prisma types
+yarn migrate:dev
```

Comment:
```
yarn migrate
```

Reply:
The reason I have two migrate scripts is in order to make difference between production and development environments.

Reply:
I mean you only created 3 migration commands: `migrate:reset`, `migrate:deploy`, `migrate` in `server/package.json`. Running `yarn migrate:dev` will result

```bash
error Command "migrate:dev" not found.
```

Reply:
Oh，yes.  Thanks @rankun203.

I have fixed the doc and add more tips about dev migrate in [7f0c9ee](https://github.com/Wiredcraft/test-fullstack/pull/78/commits/7f0c9eea5aeeca2fc4ef053b111ea7307069df20)